### ISSUE TITLE
Alias imported package to match the package name

### ISSUE THREAD

**Issue by artuross** [mvdan/gofumpt#305]
When the import path doesn't match the package name, it's a good idea to alias the import to be more explicit.

Here's an example with `github.com/testcontainers/testcontainers-go` (which defines a package `testcontainers`):

```go
import "github.com/testcontainers/testcontainers-go"
```

should be aliased to

```go
import testcontainers "github.com/testcontainers/testcontainers-go"
```

Both imports are equivalent, because in both files, package can be accessed as `testcontainers`.

On the other hand, if the package is already aliased like in the following example, `gofumpt` should not change the alias:

```go
import tc "github.com/testcontainers/testcontainers-go"
```

**Comment by mvdan** [comment]
I agree with your sentiment here - and I would imagine that gopls might already do this as part of its "organize imports" action, or it might already have a feature request for it - gofumpt works only on the syntax files so it doesn't process imported packages in any way, which would be required for this to work. That would make the tool significantly more complex and slower, and it's not really about formatting, so I think it's out of scope.
